ParabolInc/parabol
# feat(share-summary): Share summary mutation

Fixes #8233

<img width="718" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/eef08017-a32d-486e-8818-b5bc9a62abab">


How to test: 
- Follow testing steps from PR to see the dialog https://github.com/ParabolInc/parabol/pull/8331
- Share topic


Not included:

- Snackbar
